### ISSUE TITLE
Solutions dont get auto uploaded from all sections of leetcode 

### ISSUE THREAD

**Issue by disha18** [QasimWani/LeetHub#547]
## Please review previously closed issues before filling out a new one! Duplicate issues will be closed without comment.

**Describe the bug**
If I pick and solve a problem via problem list, with URL like this - https://leetcode.com/problems/maximum-product-subarray/description/
I don't see my solutions getting auto-uploaded.
However, if I solve via explore problems section example this- https://leetcode.com/explore/interview/card/google/59/array-and-strings/3053/, the submissions get auto-uploaded.

**Expected behavior**
I expect solutions to be uploaded via any URL as long as it's leetcode.

**Screenshots**
![Screenshot 2023-05-31 at 7 47 31 PM](https://github.com/QasimWani/LeetHub/assets/7753763/b89adc67-3293-4d0d-9e91-19c108114eeb)

![Screenshot 2023-05-31 at 7 49 31 PM](https://github.com/QasimWani/LeetHub/assets/7753763/c3ce7740-75e7-452c-80d5-b5ecaaffcfc1)

**Comment by 2311kamal** [comment]
@QasimWani Please look into this matter, I also have the similar problem that that my codes are not being auto uploaded in the github even I have authorized it.

**Comment by 2311kamal** [comment]
Just installed LeetHub and facing the same problem

**Comment by 2311kamal** [comment]
hey @disha18 I just found out that the leethub extension works perfectly fine with older version of leetcode.
You may switch to the older version as in the screenShot
<img width="961" alt="leetcode" src="https://github.com/QasimWani/LeetHub/assets/96480223/7ad94437-4c36-47e2-b067-a5166f5fb1c2">
Now You can see the green mark beside submit button after successfully submitting the code, and you can see the code auo pushed to github or if not refresh the page.

<img width="960" alt="Screenshot" src="https://github.com/QasimWani/LeetHub/assets/96480223/ab222485-e0d5-49f4-88ea-2892768c3824">

**Comment by disha18** [comment]
@2311kamal - thanks for taking a look. But will I not lose out on new features by reverting back to previous version? Can we not make it compatible with new version too? It’s a really useful plugin, so would love for it to be used with new version too

**Comment by 2311kamal** [comment]
Yes I know, I am also a user of leetcode like you, I look forward to see updates in this extension to use all new features of leetcode. But till then here we have a temporary solution!

**Comment by 2311kamal** [comment]
hey! @disha18 Today I encountered a new issue on leethub that is whatever code I am submitting the plugin pushes the code to github but it has stopped making the readme file that it did earlier that contains the problem description!
Are you also facing this issue?
